src/content-script/floatie/floatie.ts

```diff
@@ -317,13 +318,8 @@ export class Floatie {
       return !isNaN(Date.parse(dataStr));
     };
 
-    const hasLetters = (text: string) => {
```

Comment:
Thanks for identifying this accessibility issue and the fix! 

Instead of removing it completely, can you switch to a more robust matcher like https://stackoverflow.com/a/66808435? 

The goal of the check is to make the extension less noisy by avoiding triggering on non text selections (like when an image is selected)

Reply:
That's really my problem，I'll fix it.

Reply:
function isNotSymbols(str) {
  let notSymbols = false;
  for (let i = 0; i < str.length; i++) {
    let code = str.charCodeAt(i);
    if (
      (code >= 0 && code <= 47) ||
      (code >= 58 && code <= 64) ||
      (code >= 91 && code <= 96) ||
      (code >= 123 && code <= 255)
    ) {
      continue;
    } else {
      notSymbols = true;
      break;
    }
  }
  return notSymbols;
}
function assert(value, expected) {
    if (value === expected) {
        console.log("Pass");
    } else {
        throw new Error(`Expected ${expected}, but got ${value}`);
    }
}

assert(isNotSymbols("???"), false);
assert(isNotSymbols("a"), true);
assert(isNotSymbols("z"), true);
assert(isNotSymbols("A"), true);
assert(isNotSymbols("Z"), true);
assert(isNotSymbols("How are you?"), true);
assert(isNotSymbols("你好"), true);
assert(isNotSymbols("你好?"), true);
assert(isNotSymbols("?"), false);
assert(isNotSymbols("ÿ"), false);
assert(isNotSymbols(" 你ÿ"), true);
assert(isNotSymbols(" "), false);